api/tests/conftest.py

```diff
@@ -90,7 +94,49 @@ def dynamo_enabled_project_environment_one_document(
 @pytest.fixture()
 def dynamo_environment_wrapper(
     flagsmith_environment_table: Table,
+    settings: SettingsWrapper,
 ) -> DynamoEnvironmentWrapper:
+    settings.ENVIRONMENTS_TABLE_NAME_DYNAMO = flagsmith_environment_table.name
     wrapper = DynamoEnvironmentWrapper()
     wrapper.table_name = flagsmith_environment_table.name
     return wrapper
+
+
+@pytest.fixture()
+def app_settings_for_dynamodb(
+    settings: SettingsWrapper,
+    flagsmith_environment_table: Table,
+    flagsmith_environments_v2_table: Table,
+    flagsmith_identities_table: Table,
+) -> None:
+    settings.ENVIRONMENTS_TABLE_NAME_DYNAMO = flagsmith_environment_table.name
+    settings.ENVIRONMENTS_V2_TABLE_NAME_DYNAMO = flagsmith_environments_v2_table.name
+    settings.IDENTITIES_TABLE_NAME_DYNAMO = flagsmith_identities_table.name
+    return
+
+
+@pytest.fixture()
+def features_for_identity_clone_flag_states_from() -> (
```

Comment:
I'm not sure this fixture is really necessary and even if it is, it should live in `api/tests/conftest` it should live in the most local conftest.py file possible. 

The code, however, can be simplified to just the following, so I'd argue that we don't really need the fixture perhaps? 

```python
features = []
for i in range(1, 4): 
    features.append(Feature.objects.create(name=f"feature_{i}", project=project, default_enabled=True))
```

Reply:
Done